.env

```diff
@@ -0,0 +1,2 @@
+VALIDATOR_PRIVATE_KEY=31b571bf6894a248831ff937bb49f7754509fe93bbd2517c9c73c4144c0e97dc
+FOO=BAR
```

Comment:
unused?

Reply:
removed the file itself as we are not going ahead with .env file